browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1583,6 +1599,14 @@ void RewardsDOMHandler::SetInlineTippingPlatformEnabled(
   }
 }
 
+void RewardsDOMHandler::SetInlineTipsEnabled(const base::Value::List& args) {
+  AllowJavascript();
```

Comment:
Maybe ` CHECK_EQ(1U, args.size())` here?

Reply:
Sure! Added in [13e0469](https://github.com/brave/brave-core/pull/17339/commits/13e0469cd5d6ee5bbaf02bb3c15f34a8fa99f976)